<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeltedPlaylistDock</class>
 <widget class="QWidget" name="MeltedPlaylistDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>288</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Add something to the playlist</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove cut</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuButton">
       <property name="toolTip">
        <string>Display a menu of additional actions</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="format-justify-fill"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionInsertCut">
   <property name="text">
    <string>Insert Cut</string>
   </property>
  </action>
  <action name="actionAppendCut">
   <property name="icon">
    <iconset theme="icon-add"/>
   </property>
   <property name="text">
    <string>Append Cut</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open As Clip</string>
   </property>
   <property name="toolTip">
    <string>Open the cut in the player</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="toolTip">
    <string>Go to the start of this cut in the playlist</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="toolTip">
    <string>Remove all items from the playlist</string>
   </property>
  </action>
  <action name="actionWipe">
   <property name="text">
    <string>Wipe</string>
   </property>
   <property name="toolTip">
    <string>Remove all items before the currently playing clip</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean</string>
   </property>
   <property name="toolTip">
    <string>Remove all items except the currently playing clip</string>
   </property>
  </action>
 </widget>
 <connections/>
</ui>
